stdlib/public/runtime/Errors.cpp

```diff
@@ -69,7 +138,17 @@ reportOnCrash(const char *message)
 static void
 reportOnCrash(const char *message)
```

Comment:
Move all of this into `reportNow()`. `reportNow()` records an error to the console and/or syslog. `reportOnCrash()` records an error to other crash reporting machinery such as OS X's crash reporter.